pkg/pool-weighted/contracts/smart/IndexPool.sol

```diff
@@ -240,10 +256,11 @@ contract IndexPool is BaseWeightedPool, ReentrancyGuard, IIndexPool {
     ) public override returns (uint256) {
         //cannot swap out uninitialized token
         _require(minBalances[swapRequest.tokenOut] == 0, Errors.UNINITIALIZED_TOKEN);
+        require(_tokenState[swapRequest.tokenIn].decodeUint5(_REMOVE_FLAG_OFFSET) != _REMOVE_FLAG, "Removed token");
```

Comment:
Maybe add the error message to the `BalancerErrors.sol` so they way we show errors will be uniform

Reply:
Fixed